kuzzleio/kuzzle-device-manager
# feat: update measure model for localization

## What does this PR do ?

This PR is related to the iot-platform [PR](https://github.com/kuzzleio/iot-platform/pull/420).

It add locales for asset model and implement an endpoint to update all assets related to this asset model. 

[Ticket](https://kuzzle.atlassian.net/browse/KZLPRD-689?atlOrigin=eyJpIjoiMDU0ZDE5ZmUwODdiNDlmNzllNjMwNWViMTQ0OTg5N2MiLCJwIjoiaiJ9)



### How should this be manually tested?


  - Step 1 : Add a locales in an `asset model`
  - Step 2 : Create an `asset` from this `asset model`
  - Step 3 : Check if the locales is in the document
  - Step 4 : Update the `asset model`
  - Step 5 : Send a PATCH request to the endpoint `device-manager/assets/modelLocales`
  - Step 6 : Check the changing in all assets from the asset model

### Other changes



### Boyscout
